mercury-operator/README.md

```diff
@@ -18,3 +18,49 @@ This operator is the one responsible for the deployment of the Mercury clusters
 * On Kafka Topic successful creation, its name will be displayed in the Service Domain status.
 
 ![Mercury Operator System](images/mercury-operator.png)
+
+The Operator is Watching 2 kinds of entities:
+
+##### Service Domain Cluster
+The service domain cluster custom resource represents a cluster of service domains with a Kafka broker deployment.
+E.g.
+```yaml
+apiVersion: mercury.redhat.io/v1alpha1
+kind: ServiceDomainCluster
+metadata:
+  name: service-domain-cluster
+  labels:
+    service-domain-cluster: service-domain-cluster
+```
+
+A cluster of service domains consists of a single service domain cluster cr.
+To create a service domain cluster run the following command
+```shell
+oc create -f service-domain-cluster.yaml
+```
+
+##### Service Domain
+The service domain custom resource represents a service.
```

Comment:
I'd mention that a gRPC service will be created by default for inter service domain communication. If external access is required then use the `expose` parameter